forum/serializers.py

```diff
@@ -1,51 +1,51 @@
-# Copyright (C) 2017 Semester.ly Technologies, LLC
```

Comment:
why is this happening?

Reply:
I want to say this might be because I merged cs310 with the recent changes into my branch(?)

Reply:
@miaboloix do you think it could be the CRLF lines brought by her intelliJ settings? I remember a lot of lint errors existed because someone was using CRLF instead of LF

Reply:
Yeah it was me ;-;
Mia helped me figure it out earlier today